package.json

```diff
@@ -20,8 +20,6 @@
     ]
   },
   "resolutions": {
-    "react-native": "0.77.0",
-    "react": "18.3.1",
```

Comment:
good to keep this resolutions in case some inconsistent deps pulled into our monorepo

Reply:
added back